src/lib/storage/session.js

```diff
@@ -52,6 +53,7 @@ SessionStorage.prototype = {
   getTabs: function(chromeWindow) {
     let browser = TabsUtils.getTabBrowser(chromeWindow);
     let tabs = [];
+    let currentGroup = this.getCurrentGroup(chromeWindow);
```

Comment:
It's just a quick performance enhancement I happened to notice. The current group won't change during the loop, so there's no need to retrieve the JSON from storage over and over for each tab.